angular.json

```diff
@@ -223,6 +419,25 @@
             }
           }
         },
+        "serve-ssr-cds": {
+          "builder": "@nguniversal/builders:ssr-dev-server",
+          "options": {
+            "browserTarget": "storefrontapp:build:cds.development",
+            "serverTarget": "storefrontapp:server-cds"
+          },
+          "configurations": {
+            "production": {
+              "browserTarget": "storefrontapp:build:cds.production",
+              "serverTarget": "storefrontapp:server-cds:production",
+              "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/environment.ts",
```

Comment:
Do we need replacement in this particular config?

Reply:
Nope, it's a leftover. Thanks for catching it.